app/models/installed_technology.rb

```diff
@@ -71,7 +71,8 @@ def technology
   def behavior_with_curve(curve_type = nil)
     behavior = self.behavior.presence || technology.behavior
 
-    return behavior if curve_type.nil? || curve_type == 'default'.freeze
+    return "default" if is_edsn_profile?
```

Comment:
Makes more sense. I'm also a bit clueless when I came up with these lines of code... 
Thanks for pointing them out. I'll look into it.